flake8_trio.py

```diff
@@ -41,6 +31,79 @@ class Statement(NamedTuple):
     "TRIO110": "`while <condition>: await trio.sleep()` should be replaced by a `trio.Event`.",
 }
 
+
+class Statement(NamedTuple):
+    name: str
+    lineno: int
+    col_offset: int = 0
+
+    # ignore col offset since many tests don't supply that
```

Comment:
`and (self.col_offset == other.col_offset or None in [self.col_offset, other.col_offset])`?